File: back_end/MindEarth/MindEarth.Web/Features/Reports/DTO/DTO_ReportImage.cs
namespace MindEarth.Web.Features.Reports.DTO
{
    public class DTO_ReportImage
    {
        public string ImageName { get; set; }
        public string Figure { get; set; }
        public string ImageTitle { get; set; }
        public string SavedImageName { get; set; }

        public string RowId { get; set; }
    }

    public class DTO_CreateReportReponse
    {
        public string ReportId { get; set; }
        public List<DTO_ReportImage> Images { get; set; } = new List<DTO_ReportImage>();
    }
}

File: admin_ui/report_admin_ui/src/app/features/auth/reports-controller/models/dto-report-image.ts
export class DtoReportImage {
    figure?:string;
    imageName?:string;
    imageTitle?:string;
    savedImageName?:string;
    rowId?:string;
    isImageAvailable?:boolean;
    isImageChanged?:boolean;
    file?:File;
    imageFullPath?:string;
}


export class DtoSaveReportResonse
{
    reportId?:string;
    images?:Array<DtoReportImage> = [];
}